samples/v2/pipeline_with_secret_as_env_test.py

```diff
@@ -17,11 +17,15 @@
 
 import unittest
 
-import kfp.deprecated as kfp
+import kfp
+from kfp.samples.test.utils import KfpTask
+from kfp.samples.test.utils import run_pipeline_func
+from kfp.samples.test.utils import TaskInputs
+from kfp.samples.test.utils import TaskOutputs
+from kfp.samples.test.utils import TestCase
 import kfp_server_api
 from ml_metadata.proto import Execution
 
-from kfp.samples.test.utils import KfpTask, TaskInputs, TaskOutputs, TestCase, run_pipeline_func
 from .pipeline_secret_env import pipeline_secret_env
```

Comment:
kfp/taskinputs/taksoutputs look like they aren't used
and .pipeline_secret_env looks like it may have been removed sometime in the past